test/kubemark/start-kubemark.sh

```diff
@@ -165,7 +165,7 @@ function copy-resource-files-to-master {
 # Make startup scripts executable and run start-kubemark-master.sh.
 function start-master-components {
   echo ""
-  MASTER_STARTUP_CMD="sudo bash /home/kubernetes/start-kubemark-master.sh"
+  MASTER_STARTUP_CMD="sudo ${START_KUBEMARK_MASTER_ENVS:-} bash /home/kubernetes/start-kubemark-master.sh"
```

Comment:
I would suggest passing any such config through the master env file that we create above in `create-master-environment-file` function.